src/pages/contributing/component/index.mdx

```diff
@@ -521,19 +522,211 @@ leaving any other comments on
+
+| Element | Font-size (px/rem) | Font-weight   | Type token |
+| ------- | ------------------ | ------------- | ---------- |
+| Title   | 14 / 0.875         | Regular / 400 | `$body-01` |
+| Content | 14 / 0.875         | Regular / 400 | `$body-01` |
+
+## Structure
+
+<!--
+Use a table to document states, pacing, and any other component-specific configurations. You can reference this [table generator](https://www.tablesgenerator.com/markdown_tables) for easy table creation.
+
+Add supporting images to this section, with at least one image showing how color is applied within the component and another showing structure and spacing measurements. Images are accompanied by a descriptive caption below the image and used in conjunction with the spec charts. Note we used fixed images in the Structure section.
+
+The following table shows an example from [Accordion](https://carbondesignsystem.com/components/accordion/style).
+-->
+
+| Element | Property    | px/rem   | Spacing token |
+| ------- | ----------- | -------- | ------------- |
+| Header  | height      | 40 / 2.5 | –             |
+| Item    | border-top  | 1        | –             |
+| Title   | margin-left | 16 / 1   | `$spacing-05` |
+
+<div className="image--fixed">
+
+![Structure measurements for default accordion alignment](images/accordion-style-4.png)
+
+</div>
+
+<Caption>
+  Structure measurements for default accordion alignment. | px / rem
+</Caption>
+
+<br />
+
+<div className="image--fixed">
+
+![Spacing measurements for default accordion alignment](images/accordion-style-5.png)
+
+</div>
+
+<Caption>
+  Spacing measurements for default accordion alignment. | px / rem
+</Caption>
+
+### Variant structure
+
+<!--
+You main need to provide additional specs for component variants or default versus flush alignment. For each variant name a new section and give it a name. For example: Default alignment structure.
+-->
+
+## Modifiers
+
+<!--
+The modifiers section shows different ways of styling the component. Here are some examples:
+
+- adding an icon to a [button](https://www.carbondesignsystem.com/components/button/usage/#variations)
+- adding a button to a [notification](https://www.carbondesignsystem.com/components/notification/usage/#anatomy)
+- styling a dropdown to appear inline with other content
+- using high and low contrast styles for [notifications](https://www.carbondesignsystem.com/components/notification/usage/#high-and-low-contrast)
+- using a "light" variant in a [text input](https://www.carbondesignsystem.com/components/text-input/code)
+-->
+
+## Size
+
+<!--
+In the size section, provide a table of component size names and height. Be sure to cross-link this to the size section in the usage tab, which is where you should provide guidance about how to choose the best size for the use case.
+
+
+Example: The following table shows an example from [Button](https://carbondesignsystem.com/components/button/style).
```

Comment:
Why did you switch to button being the example here instead of accordion again (which does have sizes)?

Reply:
In a feedback session someone noted that button had a good description, and illustrated how to cross link with the usage tab sizing. Accordion doesn't have that.